packages/snowpack/src/config.ts

```diff
@@ -171,7 +171,7 @@ function expandCliFlags(flags: CLIFlags): DeepPartial<SnowpackConfig> {
       continue;
     }
     if (configSchema.properties.buildOptions.properties[flag]) {
-      result.devOptions[flag] = val;
+      result.buildOptions[flag] = val;
```

Comment:
Unrelated bugfix in another PR: CLI flags for build options currently not working.

I would add in another test, but this was needed for this test.

Reply:
works for me 👍